Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -39,9 +39,13 @@ public bool isServer
         public bool isLocalPlayer { get; private set; }
         public bool hasAuthority { get; private set; }
 
+        // observers: all the player connections that we broadcast to
         // <connectionId, NetworkConnection>. always empty on clients.
         public Dictionary<int, NetworkConnection> observers = new Dictionary<int, NetworkConnection>();
 
+        // visList: all objects that broadcast to us
+        public readonly HashSet<NetworkIdentity> visList = new HashSet<NetworkIdentity>();
```

Comment:
can you make it internal?   that way if we move it again in the future we don't have to obsolete it

Reply:
people might need it from the outside. it was public in networkconnection too.